templates/partials/actionbar.html

```diff
@@ -105,7 +105,7 @@
               <span class="fa fa-pencil"></span>
               <p translate>Edit</p>
             </a>
-            <a class="mm-icon mm-icon-inverse mm-icon-caption" ng-click="actionBar.disableDelete || deleteDoc(actionBar.selected)" ng-class="{'mm-icon-disabled': actionBar.disableDelete}" mm-auth="can_delete_reports">
```

Comment:
This isn't needed any more - the CSS blocks interaction with the element when the `mm-icon-disabled` class exists.